src/lib/check-failing.js

```diff
@@ -7,8 +7,23 @@ const the = {
 	failures: []
 };
 
-module.exports.init = function () {
+let getTimezone = () => {
+	if (process.env.TZ) {
+		return process.env.TZ;
+	}
+
+	if (fs.existsSync('/etc/timezone')) {
+		return fs.readFileSync('/etc/timezone');
+	}
+
+	if (fs.lstatSync('/etc/localtime').isSymbolicLink()) {
+		return fs.readlinkSync('/etc/localtime').replace('/usr/share/zoneinfo/', '');
+	}
+
+	console.error('Error occurred when evaluating the system timezone'); // eslint-disable-line
```

Comment:
I'd replace with:
`logger.error({ event: 'EXPRESS_GET_TIMEZONE_FAIL' });`
then return `new Date().getTimezone()` (or whatever is the fix in my PR)